src/response/chi0.jl

```diff
@@ -346,20 +346,25 @@ end
     ψ_extra = [ψ[ik][:, maskk] for (ik, maskk) in enumerate(mask_extra)]
 
     ε_occ   = [eigenvalues[ik][maskk] for (ik, maskk) in enumerate(mask_occ)]
+    δHψ_occ = [δHψ[ik][:, maskk] for (ik, maskk) in enumerate(mask_occ)]
 
     occ_occ = [occ[ik][maskk] for (ik, maskk) in enumerate(mask_occ)]
 
-    # First we compute δoccupation and δεF
-    δocc, δεF = compute_δocc(basis, ψ_occ, occ_occ, εF, ε_occ, δHψ)
-    # Pad δoccupation
+    # First we compute masked δoccupation and δεF…
```

Comment:
delta occupation and eps f for the occupied orbitals (the mask is just the technical way we select the occupied orbitals), same below